.eslintrc.json

```diff
@@ -26,25 +26,44 @@
     "jsx-a11y/no-noninteractive-element-interactions": "off",
     "jsx-a11y/no-static-element-interactions": "off",
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-    "quotes": ["error", "double"],
+    "quotes": ["error", "double", { "allowTemplateLiterals": true }],
+    "allowTemplateLiterals": true,
     "parserOptions": {
       "allowImportExportEverywhere": true,
       "sourceType": "module"
     },
     "eol-last": ["error", "always"],
     "import/prefer-default-export": "on",
     "import/resolver": {
-      "node": { "extensions": [".js"] }
+      "node": {
+        "extensions": [".js"]
+      }
     },
     "one-var": 0,
     "one-var-declaration-per-line": 0,
     "new-cap": 0,
     "consistent-return": 0,
     "no-param-reassign": 0,
     "comma-dangle": 0,
+    "global-require": "off",
```

Comment:
By putting  off for this rule , you mean you disable the use of the global require in the files?

Because using global require can lead to perfrmance issues in the future as said here: 

```Since require() does a synchronous load, it can cause performance problems when used in other locations.```

Reply:
So, what are you suggesting on this?